web/client/src/library/components/fileExplorer/context.tsx

```diff
@@ -125,9 +125,19 @@ export default function FileExplorerProvider({
       })
   }
 
-  function createFile(parent: ModelDirectory, extension = '.py'): void {
+  function createFile(parent: ModelDirectory, extension?: string): void {
     if (isLoading) return
 
+    if (isNil(extension)) {
+      if (parent.isModels) {
+        extension = '.sql'
```

Comment:
I guess most users would want SQL models, but we do have Python models.

Reply:
It just default, user can rename file to get python model. If mostly users be creating sql and occasionally python it make sense to set default to sql